src/graph/backend/elyzor/include/dnnl_graph_compiler.h

```diff
@@ -29,7 +29,7 @@ struct dnnl_graph_compiler_context {
 struct dnnl_graph_compiler_tensor {
     size_t id;
     uint8_t ndims;
-    size_t *dims;
+    int64_t *dims;
```

Comment:
_Originally posted by @dchigarev in https://github.com/intel-ai/oneDNN/pull/2#discussion_r1588893219_

size_t -> int64_t so we can reuse dims buffer of oneDNN's tensors without reinterpret cast